oe_theme.theme

```diff
@@ -22,6 +22,7 @@ function oe_theme_preprocess(&$variables) {
   global $base_url;
   $theme = \Drupal::theme()->getActiveTheme();
   $variables['ecl_icon_path'] = $base_url . '/' . $theme->getPath() . '/dist/images/icons/sprites/icons.svg';
+  $variables['ecl_social_icon_path'] = $base_url . '/' . $theme->getPath() . '/dist/images/social-icons/sprites/icons-social.svg';
```

Comment:
Could we do that as there: https://github.com/openeuropa/oe_theme/pull/256/files#diff-42cf9094a83a0529c3f942ad0d192503R24?